bootstrap/092_service_account_setup.sql

```diff
@@ -25,7 +25,8 @@ begin
 
     -- Bind the given reference ID to the 'OPSCENTER_API_INTEGRATION' reference. Must match the reference in manifest.yml
     -- differs from v1 in that all external functions are not re-created by update_reference()
-    call admin.update_reference('OPSCENTER_API_INTEGRATION', 'finalize_setup_from_service_account', :api_integration_ref_id);
+   insert into internal.reference_management (ref_name, operation, ref_or_alias) values ('OPSCENTER_API_INTEGRATION', 'Running external functions setup proc.', :api_integration_ref_id);
+    call admin.connect_sundeck(:token);
```

Comment:
`connect_sundeck` returns an object which contains some specific error information. It would be good to change this proc to `RETURNS OBJECT` and return the result from `admin.connect_sundeck`

Reply:
done